src/styles/base/_typography.scss

```diff
@@ -165,6 +167,7 @@ pre[ class*="language-" ] {
 	border-radius: 2px;
 }
 
-::selection {
+::selection,
+pre::selection {
```

Comment:
Looks like the `pre::selection` stuff from prism.js was removed, so is this still needed? If it is, do we need `code::selected` too?